hooks/boot/01-kafo-hook-extensions.rb

```diff
@@ -0,0 +1,11 @@
+module HookContextExtension
+  def success_file
+    File.join(File.dirname(Kafo::KafoConfigure.config_file), '.installed')
```

Comment:
I think this should also work and be cleaner in testing:

```suggestion
    File.join(File.dirname(kafo.config.config_file), '.installed')
```

Ideally we'd avoid having a `.installed` file. I'd actually consider storing it in the scenario itself. However, I won't block on that.